src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => {
+    return century !== undefined
+      ? person.sex === 'm' && century === Math.ceil(person.died / 100)
+      : person.sex === 'm';
+  });
+
+  const totalAge = men.reduce((sum, item) => {
```

Comment:
Think should parameter `item` should be renamed to a more descriptive name like `person`, please do that for the rest of functions in the projects